src/CMakeLists.txt

```diff
@@ -89,6 +89,7 @@ if (USE_ANGELSCRIPT)
             ${CMAKE_SOURCE_DIR}/external/as_add_on/scriptstdstring/scriptstdstring.h
             ${CMAKE_SOURCE_DIR}/external/as_add_on/scriptstdstring/scriptstdstring.cpp
             Scripting/ScriptEngine.{hxx,cxx}
```

Comment:
if (USE_ANGELSCRIPT) ?

Reply:
Yeah, throughout the code we have that as a flag because you can toggle it during the build.